### ISSUE TITLE
Add a biome check

### ISSUE THREAD

**Issue by Emmet-v15** [Gaider10/CrossroadFinder#5]
It's useful if you want a structure in a specific biome

**Comment by BalancedOneshot** [comment]
True, most people will just world eater but for an early game farm it would be nice to specify soul sand valley.

**Comment by Emmet-v15** [comment]
adding to this, if this were to become a thing a 256x256 perimeter of the biome would be adequate (although allowing custom values would be nice too)
someone else has this feature, although that only checks for normal fortresses, not crossroads
